### DISCUSSION TITLE
ValueError: Not a boolean: y

### DISCUSSION THREAD

**Discussion by ckbisek** [Jip-Hop/jailmaker#194] [Q&A] [answered]
While I was successful in creating a jail, I cannot perform a list or startup.  Reference screenshot for error.  Thank you

![image](https://github.com/Jip-Hop/jailmaker/assets/47727235/609150c1-bb7a-45a2-8390-c22a268d676b)

**Comment by Jip-Hop** [comment, marked as answer]
Please post your config file. Probably you're not using 0 and 1.

**Reply by ckbisek** [reply]
That was it!  The very first line of my jail had
`startup=y`

Not sure where I got that from, but all is well after I changed 'y' to '1'

Thank you!!
